src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -209,6 +206,14 @@ static boolean passesBasicCvrSourceValidation(CvrSource source) {
           Logger.log(Level.SEVERE, "overvoteDelimiter is invalid.");
         }
       } else {
+        if (provider == Provider.CDF) {
+          if (!source.getFilePath().endsWith(".xml") && !source.getFilePath().endsWith(".json")) {
+            Logger.log(Level.SEVERE,
+                "Unexpected file extension: %s.  CDF source files must be .xml or .json",
```

Comment:
double space

Reply:
Yeah, I was secretly trying to get him to fix this by suggesting this haha:
```
Logger.severe("CDF source files must be .json or .xml! Unexpected file extension for: %s", source.getFilePath());
```